src/content/en/2022/fonts.md

```diff
@@ -1,22 +1,851 @@
+
+We hope to see more use of these generic family names next year. They are great for UI elements, forms, or really anything where you want to evoke a native feel. As an added benefit, they are also great for performance as they are guaranteed to use a locally installed font.
+
+### Font smoothing
+
+{{ figure_markup(
+  image="usage-of-font-smoothing-properties.png",
+  caption="Usage of font smoothing properties.",
+  description="Bar chart showing `-webkit-font-smoothing: antialiased` is used on 74% of desktop and 73% of mobile pages, `-moz-osx-font-smoothing: grayscale` on 67% and 66%, `-webkit-font-smoothing: auto` on 13% and 12%, and finally `-webkit-font-smoothing: subpixel-antialiased` is used on 13% of desktop and 13% of mobile pages..",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQF6OH_2-0apcFzjE-iHSQNuZqp9DtM7udIeOzPSOSMM-Pf6KdTnRwAclX9QPZF1vNNgu6acZvqoN5b/pubchart?oid=534216530&format=interactive",
+  sheets_gid="1970926839",
+  sql_file="font_smoothing_usage.sql"
+  )
+}}
+
+And now for a complete surprise—to us anyway: people really like to specify their MacOS-only [font smoothing preferences](https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth). For example, the `-webkit-font-smoothing: antialiased` value is used on 73.4% of all sites. This is surprising because it—and its siblings `-mox-osx-font-smoothing`, and `font-smoothing`—are not even official CSS properties. This might make them the most used unofficial CSS properties!
+
+Our hunch this is a combination of CSS frameworks including these properties and a dislike of how fonts are rendered slightly bolder on macOS—variable font grades to the rescue! It would be interesting to return to these properties in the 2023 Fonts chapter. Perhaps it is also time to put these properties on a standards track? The demand is clearly there.
+
+## Variable fonts
+
+_Variable fonts_ allow type designers to combine multiple styles of a family into a single font file. They do this by defining one or more design axes, such as weight (thin, regular, and bold) or width (condensed, normal, and expanded). Instead of storing each style as individual font files, a variable font interpolates them from a handful of "master" instances.
+
+For example, even if the type designer did not explicitly create a semibold style, using a variable font with a weight axis, the text rendering engine will simply interpolate a semibold style for you (and any other weight you might need, assuming the variable font's weight axis supports that range). Not only do variable fonts increase typographic expressiveness, they also offer a major benefit for web developers in terms of file size savings when several font variations are used. Variable fonts will however be larger than a single variation.
```

Comment:
I added a caveat at the end of this paragraph, as it's a bit of a hot topic in webperf community.

Reply:
👍